src/bootstrap/dist.rs

```diff
@@ -1852,6 +1954,9 @@ impl Step for Extended {
             if rls_installer.is_some() {
                 prepare("rls");
             }
+            if rust_analyzer_installer.is_some() {
```

Comment:
Let's make the Step return just `PathBuf` instead of Option so that we can avoid these checks